app/tasks/summary_tasks.py

```
@@ -129,15 +320,19 @@ def generate_batch_summaries_task(
 
         # GPU 클라이언트 초기화
         from app.clients.summary_client import SummaryClient
-        summary_client = SummaryClient(timeout=600)  # 10분 타임아웃
+        from app.core.exceptions import SummaryServerException, GPUTimeoutException
```

Flake8 (7.3.0)

[error] 'app.core.exceptions.SummaryServerException' imported but unused

(F401)